pkg/ignite/create.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
I don't think this copyright should be here, as far as I can see it's not existent in the other files. please remove